package.json

```diff
@@ -4,20 +4,61 @@
   "displayName": "Games Portal",
   "cordova": {
     "platforms": [
-      "android"
+      "android",
+      "ios"
     ],
     "plugins": {
       "cordova-plugin-contacts": {
-        "CONTACTS_USAGE_DESCRIPTION": "Games Portal wants to use your contacts"
```

Comment:
keep

Reply:
My bad, I think when I added dependencies/did the setup for iOS, it over wrote some of hers, I'll add them back in!